src/com/google/enterprise/adaptor/DocumentHandler.java

```diff
@@ -1252,8 +1267,12 @@ private void transformMetadata() {
       }
       crawlOnce = Boolean.parseBoolean(params.get(KEY_CRAWL_ONCE));
       lock = Boolean.parseBoolean(params.get(KEY_LOCK));
-      // TODO: make constants for this growing set of valid keys
-      considerNotSending(params.get(KEY_TRANSMISSION_DECISION), docId);
+      String transmissionDecision
+          = params.get(KEY_FORCED_TRANSMISSION_DECISION);
```

Comment:
We are leaving open the possibility of supporting the equivalent of --force for filters to set FORCED_TRANSMISSION_DECISION instead of TRANSMISSION_DECISION.  We will try to keep you from shooting yourself in the foot, but that might be exactly what some customer wants to do.